tika/server.go

```diff
@@ -67,14 +103,25 @@ func NewServer(jar, port string) (*Server, error) {
 	return s, nil
 }
 
+// ChildMode sets up the server to use the -spawnChild option.
+// If used, ChildMode must be called before starting the server.
+// If you want to turn off the -spawnChild option, call Server.ChildMode(nil).
+func (s *Server) ChildMode(ops *ChildOptions) error {
+	if s.cmd != nil {
+		return fmt.Errorf("Server Process already started, cannot switch to spawn child mode")
```

Comment:
```suggestion
		return fmt.Errorf("server process already started, cannot switch to spawn child mode")
```

https://github.com/golang/go/wiki/CodeReviewComments#error-strings